docs/LitActions/usingPKPsAndActions.md

```diff
@@ -22,6 +22,10 @@ As a distributed key management network, Lit provides developers with the abilit
 
 A user can create a new PKP and grant a Lit Action the right to sign using it. This means the distributed key has the ability to sign and decrypt arbitrary data based on pre-defined logic and conditions.
 
+### Adding a Permitted Address
+
+You can use the [PKPPermissions contract](https://github.com/LIT-Protocol/LitNodeContracts/blob/main/contracts/PKPPermissions.sol#L418) to add additional permitted auth methods and addresses to your PKP. Note that these users will be able to execute any permitted Lit Actions and functionality associated with that PKP. 
+
```

Comment:
It's not just signing for permitted LA, they could add the PKP as a signer to additional LA's. 

Maybe broader language around this like, "Permitted parties are granted permission to sign with the PKP"